packages/flutter_tools/lib/src/migrations/cmake_policy_migration.dart

```diff
@@ -0,0 +1,59 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import '../base/file_system.dart';
+import '../base/project_migrator.dart';
+import '../cmake_project.dart';
+
+// Migrate CMake's policy to CMP0135.
+// See https://github.com/flutter/flutter/issues/116866.
+class CmakePolicyMigration extends ProjectMigrator {
```

Comment:
Should we migrate all the affected `CMakeLists.txt` in this repo? Otherwise `flutter run` will modify these files.

Reply:
Can you explain what modifications you expect `flutter run` to make to which files?

Reply:
Your change causes `flutter run` to migrate an app's `./windows/CMakeLists.txt` file if it uses the old CMake policy. This repo has many apps that use the old CMake policy: https://github.com/search?q=repo%3Aflutter%2Fflutter+cmake_policy%28SET+CMP0063+NEW%29+path%3A**%2FCMakeLists.txt&type=code

Reply:
That link is not showing any results for me, but if you mean to check in the migration applied to existing/example apps in our repo, then good call